python_modules/libraries/dagster-k8s/dagster_k8s/ops/k8s_job_op.py

```diff
@@ -416,6 +416,16 @@ def execute_k8s_job(
             num_pods_to_wait_for=num_pods_to_wait_for,
         )
     except (DagsterExecutionInterruptedError, Exception) as e:
+        try:
+            pods = api_client.get_pod_names_in_job(job_name=job_name, namespace=namespace)
+            pod_debug_info = "\n\n\n\n".join([api_client.get_pod_debug_info(pod_name, namespace) for pod_name in pods])
```

Comment:
Can you change "\n\n\n\n" to "\n"?

Reply:
This would reduce the visibility, because `get_pod_debug_info` itself returns `\n\n`, so I think the lowest we can go is `\n\n`, but I proposed double that to visually distinguish between pod debug info of different pods.
Alternatively we could also log separately for each pod, but as I understood from Your previous comments, that's not something You'd encourage.

What do You think would the the most reasonable solution here? Shall we stick to `\n\n\n\n` or modify it?

Reply:
I was going off of the similar code here: https://github.com/dagster-io/dagster/blob/master/python_modules/libraries/dagster-k8s/dagster_k8s/launcher.py#L379-L390

Maybe that should be "\n\n".join too?

I think "\n\n" is fine - that creates a single line of whitespace between each pod debug info right?

Reply:
yup, it's a single line in between.

I'm not sure how Your reference looks like, but I could add `\n` there as well if it makes sense. Actually, initially I was looking for ways to reuse this code, but couldn't figure our how. Looks like k8s is using different launcher, at least what I figured out from debugging the inheritance. So I decided to keep things simple and avoid refactoring wormhole. :)